blazor/rating/selection.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title: Selection in Blazor Rating Component | Syncfusion
+description: Checkout and learn here all about Selection with Syncfusion Blazor Rating component in Blazor Server App and Blazor WebAssembly App.
+platform: Blazor
+control: Rating
+documentation: ug
+---
+
+# Selection in Blazor Rating Component
+
+All the items before the selected item should be in selected state in a rating component.
```

Comment:
Don't use should, as if it's a requirement. Just brief the selection behavior.

Reply:
I have updated it as per comments.